tests/db_functions/test_datetime.py

```diff
@@ -147,9 +147,14 @@ def test_extract_func(self):
         with self.assertRaisesMessage(ValueError, 'lookup_name must be provided'):
             Extract('start_datetime')
 
-        with self.assertRaisesMessage(ValueError, 'Extract only valid on Date/Time/DateTimeFields'):
+        with self.assertRaisesMessage(
+                ValueError, 'Extract input expression must be one of DateField, DateTimeField or TimeField.'):
```

Comment:
I usually write `msg = '...'`on the line before `with self.assertRaisesMessage(ValueError, msg):` to avoid the unusual indentation.